prime-router/docs/universal-pipeline/README.md

```diff
@@ -1,24 +1,272 @@
+- **data formats** - At present, the pipeline can receive and send *HL7v2.5.1* and *FHIR* formats
+- **data types** - While presently only handling the routing of ELR (Laboratory Data), The pipeline was designed to be able to process the [six core data sources](https://www.cdc.gov/ophdst/public-health-data-strategy/phds-core-data-sources.html) as identified by PHDS:
+    - Case data
+    - Laboratory data
+    - Emergency department data
+    - Vital statistics data
+    - Immunization data
+    - Healthcare capacity and utilization data
 
-## Data Pipeline through ReportStream
-([useful reference](https://docs.google.com/document/d/1srnzARL_OZDm_-Rm9GyJjVVXmtPdOWscSgcBGep3dnQ/edit))
-- Include graphic of the pipeline phases (Receive, Convert, Route, Translate, Batch, Send) as well as the full architecture diagram
-- Metadata Store
-- Blob Store - include a sub-section for each folder, especially the receive and ready folders.
-- Receiver’s server
-- Lineage Tracking
-  - Report lineage
-  - Item lineage
-- Asynchronous Pipeline (explain how this affects the history query)
+The rest of this document aims to explore the high level architecture of the Universal Pipeline and how exactly it can meet the promises outlined above!
 
-## Data Retention Policy
+## Data Pipeline
+In order to handle different data **formats** and **types** in a scalable and maintainable way, the pipeline has been broken up into the following independent "steps":
 
-## Pipeline steps
+```mermaid
+flowchart LR;
+    Receive-->Convert;
+    Convert-->Route;
+    Route-->Translate;
+    Translate-->Batch;
+    Batch-->Send;
+```
+The sections listed below will aim to describe each step of the pipeline in technical detail. Each step will refer to its part of the detailed architecture diagram found [here](https://lucid.app/lucidchart/invitations/accept/inv_daa20a3a-7979-4463-8932-26033c55abb3). See [Pipeline Steps Overview](#pipeline-steps-overview) for a high level overview of how the different steps combine to form the core functionality of the Universal Pipeline.
 
 - [Receive](./receive.md)
 - [Convert/Translate](./convert-translate.md)
 - [Route](./route.md)
 - [Batch](./batch.md)
-- [Send](./send.md)
\ No newline at end of file
+- [Send](./send.md)
+
+### Pipeline Scalability and Architecture
+Each pipeline step runs as its own *Azure Function* (equivalent to lambdas in AWS) in ReportStream's Microsoft Azure Account. This means ReportStream and its Universal Pipeline are architected for, and deployed to, the cloud. The Universal Pipeline is horizontally and vertically scalable. AWS manages the life cycle of each function and dynamically decides how many of each function to instantiate based on load - ensuring ReportStream can handle even peak COVID reporting numbers!
```

Comment:
```suggestion
Each pipeline step runs as its own *Azure Function* (equivalent to lambdas in AWS) in ReportStream's Microsoft Azure Account. This means ReportStream and its Universal Pipeline are architected for, and deployed to, the cloud. The Universal Pipeline is horizontally and vertically scalable. Azure manages the life cycle of each function and dynamically decides how many of each function to instantiate based on load - ensuring ReportStream can handle even peak COVID reporting numbers!
```

Reply:
Ha, good catch! Looks like my subconscious really wants to use AWS